update.sh

```diff
@@ -35,20 +37,47 @@ for version in '.'; do
 	linuxArchCase+=$'\t\t''*) echo >&2 "error: current architecture ($dpkgArch) does not have a corresponding Julia binary release"; exit 1 ;; '$'\\\n'
 	linuxArchCase+=$'\t''esac'
 
+	winSha256="$(echo "$sha256s" | grep "*julia-${fullVersion}-win64.exe$" | cut -d' ' -f1)"
+
 	echo "$version: $fullVersion"
 
-	sed -r \
-		-e 's!%%JULIA_VERSION%%!'"$fullVersion"'!g' \
-		-e 's!%%ARCH-CASE%%!'"$(sed_escape_rhs "$linuxArchCase")"'!g' \
-		Dockerfile-debian.template > "$version/jessie/Dockerfile"
+	for v in \
+		{jessie,stretch} \
+		windows/windowsservercore-{1709,ltsc2016} \
+	; do
+		dir="$version/$v"
+		variant="$(basename "$v")"
+
+		mkdir -p "$dir"
+
+		case "$variant" in
+			windowsservercore-*) template='windowsservercore'; tag="${variant#*-}" ;;
+			*) template='debian'; tag="$variant" ;;
```

Comment:
I thought about adding `-slim` to `tag` here -- thoughts?

(This would switch us from `debian:jessie` to `debian:jessie-slim`, removing man pages and unnecessary documentation from the base and other Debian packages, thus resulting in a smaller image.)

Reply:
Maybe that can be done in a separate PR.